hs_discover/src/router.js

```diff
@@ -1,17 +0,0 @@
-import Vue from 'vue';
```

Comment:
urls.py handle paths, a future project should use NGINX with SPA or SPA-like features/routing with maybe a flask backend